java/ddlogapi.c

```diff
@@ -444,6 +444,11 @@ JNIEXPORT void JNICALL Java_ddlogapi_DDlogAPI_00024DDlogCommandVector_ddlog_1bat
     ddlog_free_record_updates((ddlog_record*)handle, size);
 }
 
+JNIEXPORT void JNICALL Java_ddlogapi_DDlogAPI_ddlog_1free(
```

Comment:
where is this used?

Reply:
It's used in dispose.

Reply:
Well, dispose is a wrapper around release, which is why I was confused. `release` was not touched by this PR, so it must already have access to `ddlog_free` if it needed it?

Reply:
release was actually never called by anyone.
This is the first use of it.
There should not be both of them, but I wanted to keep backwards compatibility.